model/src/main/kotlin/config/Curations.kt

```diff
@@ -21,10 +21,18 @@ package org.ossreviewtoolkit.model.config
 
 import com.fasterxml.jackson.annotation.JsonInclude
 
+import org.ossreviewtoolkit.model.PackageCuration
+
 /**
  * Curations for artifacts in a repository.
  */
 data class Curations(
+    /**
+     * Curations for third-party packages.
+     */
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    val packageCurations: List<PackageCuration> = emptyList(),
```

Comment:
For consistency I believe it should be named `packages` which is a reserved word.
How about naming it `packagez` or `pkgs` and override the serialized name.

Reply:
Only `package` is a reserved word. The plural is not. I'll change it.